pkg/querier/queryrange/queryrangebase/split_by_interval.go

```diff
@@ -1,4 +1,4 @@
-package queryrange
+package queryrangebase
 
 import (
 	"context"
```

Comment:
Not sure we need this file ?

Reply:
Just checked and yes, we need it.

Reason is, although this same implementation is available under `queryrange`, since `queryrange` imports `queryrangebase`, `queryrangebase` can't import `queryrange` (would lead to import cycle) and `queryrangebase` also needs this.